ui/src/app/domains/views/DomainsList/DomainsTable/DomainsTable.tsx

```diff
@@ -113,6 +129,7 @@ const DomainsTable = (): JSX.Element => {
 
   return (
     <MainTable
+      className="p-table-expanding--light"
```

Comment:
@ClementChaumel I figured out that this class was required for the confirmation to have the white background.

Reply:
Nice one!